### PR TITLE
Change website to use UTF-8 encoding consistently

### PR DESCRIPTION
**Pull request by fingolfin**
Resolves issue #6

Require https://github.com/gap-system/Mixer/pull/3

### CODE REVIEW

**Comment by fingolfin** [comment]
Motivation for doing this now, instead of waiting for a website rewrite: Well, that website rewrite may never happen, or in 10 years, or next month -- the latter being most unlikely of the three options. In the meantime, editing these files while they are not in UTF-8 becomes more and more of a hassle, and already several mistakes crept into the site (this PR fixes some of them, I may have missed more, though I did try to be very careful in converting everything).

**Comment by alex-konovalov** [comment]
Looks good - have you performed a test of both related changes here and in the Mixer, trying to build the site locally?

**Comment by fingolfin** [comment]
Yes I did.

File: Faq/faq.mixer [hunk 1]
```diff
@@ -51,7 +51,7 @@
         <a href="#2.1">2.1: Where can I download <mixer var="GAP"/></a>?
       </li>
       <li>
-        <a href="#2.2">2.2: I cannot FTP that much data. Can I get a CD?</a>
+        <a href="#2.2">2.2: I (obsolete question, removed)</a>
```
Comment (alex-konovalov) [inline Faq/faq.mixer]: Please see my comment at https://github.com/gap-system/GapWWW/pull/73

**Comment by fingolfin** [comment]
Oops, that FAQ change wasn't even supposed to be in this PR. Removed it now!

**Pull request merged**